pipelines/wdl/arrays/imputation_beagle/input_qc/test_inputs/Plumbing/NA12878_x10_hg38_arrays_qc_success.json

```diff
@@ -0,0 +1,8 @@
+{
+    "ArrayImputationQC.multi_sample_vcf": "gs://broad-gotc-test-storage/imputation_beagle/input_qc/scientific/vcfs/NA12878_10_duplicate.merged.cleaned.vcf.gz",
```

Comment:
probably dont need this scientific test in the plumbing section

Reply:
there sort of isn't a difference between scientific and plumbing tests here. happy to remove this one from plumbing since it is also in the scientific section but it's the only plumbing one that has a full genome input that succeeds.